modules/stock/commerce_pos_stock.module

```diff
@@ -41,3 +41,40 @@ function commerce_pos_stock_form_commerce_pos_return_alter(&$form, &$form_state,
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function commerce_pos_stock_form_commerce_pos_exchange_alter(&$form, &$form_state, $form_id) {
+  if ($form['#transaction_type'] == 'exchange') {
+    $order_wrapper = !empty($form_state['order_wrapper']) ? $form_state['order_wrapper'] : FALSE;
+    if (!empty($order_wrapper->commerce_line_items)) {
+      foreach ($order_wrapper->commerce_line_items as $line_item_wrapper) {
+        // Only show product line item types.
+        if ($line_item_wrapper->type->value() != 'product') {
+          continue;
+        }
+
+        $line_item_id = $line_item_wrapper->line_item_id->value();
+        $line_item_element = &$form['line_items'][$line_item_id];
+
+        $line_item_element['stock_wrapper_open'] = array(
+          '#markup' => '<div class="stock-wrapper line-item-col">',
```

Comment:
Again, it might make more sense to fix it in both places. But I don't think this one matters as much as the other.